### PR TITLE
Added NuGet profiles to remove when creating a release

### PR DESCRIPTION
**Pull request by flagbug**
There are a few more profiles that can be safely removed, e.g Xamarin and Windows Phone, I've also added the remaining abbreviations for WinRT

### CODE REVIEW

File: src/Shimmer.Core/ReleasePackage.cs [hunk 1]
```diff
@@ -118,15 +118,15 @@ void removeDeveloperDocumentation(DirectoryInfo expandedRepoPath)
                 .ForEach(x => x.Delete());
         }
 
-        void removeSilverlightAssemblies(DirectoryInfo expandedRepoPath)
+        void removeUnrelatedAssemblies(DirectoryInfo expandedRepoPath)
```
Comment (anaisbetts) [inline src/Shimmer.Core/ReleasePackage.cs]: Hm, not so fond of the name "Unrelated". Maybe "NonDesktop"?

File: src/Shimmer.Core/ReleasePackage.cs [hunk 2]
```diff
@@ -118,15 +118,15 @@ void removeDeveloperDocumentation(DirectoryInfo expandedRepoPath)
...
-            // profiles can be removed because you can load net20 DLLs inside 
-            // .NET 4.0 apps
+            // NB: Nuke Silverlight, WinRT, WindowsPhone and Xamarin assemblies. 
+            // We can't tell as easily if other profiles can be removed because 
+            // you can load net20 DLLs inside .NET 4.0 apps
             var libPath = expandedRepoPath.GetDirectories().First(x => x.Name.ToLowerInvariant() == "lib");
             this.Log().Debug(libPath.FullName);
 
-            var bannedFrameworks = new[] {"sl", "winrt", "netcore", };
+            var bannedFrameworks = new[] {"sl", "winrt", "netcore", "win8", "windows8", "MonoAndroid", "MonoTouch", "MonoMac", "wp", };
```
Comment (anaisbetts) [inline src/Shimmer.Core/ReleasePackage.cs:129]: Have you checked to make sure this doesn't pick up PCL profiles like `netcore45+net45+wp8`?

Comment (flagbug) [inline src/Shimmer.Core/ReleasePackage.cs:129]: PCL profiles start with "portable", so this should work

File: src/Shimmer.Core/ReleasePackage.cs (hunk 1 above)
Comment (flagbug) [inline src/Shimmer.Core/ReleasePackage.cs]: Yep, changed that in the latest commit :+1:

**Comment by shiftkey** [comment]
Looks good, would love an integration test around this to demonstrate how the folders are nuked.

File: src/Shimmer.Core/ReleasePackage.cs (hunk 2 above)
Comment (anaisbetts) [inline src/Shimmer.Core/ReleasePackage.cs:129]: :cool:

**Comment by flagbug** [comment]
@shiftkey I've added the additional profiles to the existing integration test

**Comment by anaisbetts** [comment]
Thanks @flagbug!

**Pull request merged**